app/javascript/multilang_de.css

```
@@ -0,0 +1,3 @@
+:root[lang="de"] .multilang:not([lang="de"]) {
+	display: none;
```

We don't have a linter for CSS yet, so that I would appreciate swapping the `tab` indention with 2 `spaces`.

```suggestion
  display: none;
```